lib/erb_lint/linters/rubocop.rb

```diff
@@ -147,7 +147,8 @@ def resolve_inheritance(hash, inherit_from)
       def base_configs(inherit_from)
         regex = URI::DEFAULT_PARSER.make_regexp(%w(http https))
         configs = Array(inherit_from).compact.map do |base_name|
-          if base_name.match?(/\A#{regex}\z/)
+          # rubocop:disable Performance/RegexpMatch
```

Comment:
I think you should rather you change `TargetRubyVersion: 2.4` to ``TargetRubyVersion: 2.3` in .rubocop.yml.

Reply:
You are completely right.